docs/_sections/dv_transfer_station.md

```diff
@@ -1,45 +1,9 @@
 ---
 layout: post
-title: DV Transfer Station
+title: Hardware Setup 
```

Comment:
Do you want to rename the file too?

Reply:
not for the time being at least! makes it easier to tie into the non-online docs

Reply:
:+1: